index.bs

```diff
@@ -2287,6 +2288,10 @@ same as the [=window handle=].
 Each [=/browsing context=] also has an <dfn>associated storage partition</dfn>,
 which is the [=storage partition=] it uses to persist data.
 
+Each [=/browsing context=] also has an <dfn>original opener</dfn>, which
```

Comment:
While reading this paragraph, it makes you think that there is always an opener, which is not the case.
Something like below should be more accurate 
```
is a [=/browsing context=] that opened the browsing context or null.